src/components/myAccounts/CriminalCheck.tsx

```diff
@@ -83,6 +86,17 @@ export const CriminalCheck: React.FC<CriminalCheckProps> = ({
                                 date: convertToShortDateString(submitDate, locale.en, true),
                             })}
                         </Text>
+                        <Flex>
+                            <Text pr={1}>
+                                {t("account.document", {
+                                    ns: "common",
+                                })}
+                                :{" "}
+                            </Text>
+                            <ChakraLink href={docLink} isExternal>
+                                {link}
```

Comment:
This is the file name correct? If so, might be good to rename just to make it more clear, esp since in `useFileRetrieve`, the parameter corresponds to `file`. This is being pretty nitpicky though :P

Reply:
yeaa true. It's just that it's named the "link" in our schema so we used to pass this "link" around before. nowadays, the "link" value is just the name of the file that was uploaded.